test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -68,7 +68,9 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", Ordered, func() {
 		profile, err = profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
 		Expect(err).ToNot(HaveOccurred())
 
-		By(fmt.Sprintf("Checking the profile %s with cpus %s", profile.Name, cpuSpecToString(profile.Spec.CPU)))
+		cpus, err := cpuSpecToString(profile.Spec.CPU)
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
good, but let's make sure these expectations have annotation **when needed** to make sure the failure is clear on error

Reply:
Done